resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -507,6 +512,12 @@ protected void doResolve(String inetHost,
             return;
         }
 
+        if (inetHost.isEmpty()) {
```

Comment:
@Scottmitch yes it does but not our DefaultNameResolver where we disallow null: https://github.com/netty/netty/blob/4.1/resolver/src/main/java/io/netty/resolver/SimpleNameResolver.java#L58

We could remove this restriction tho... WDYT ?